Tests broken since last update

Since the last update, the tests are broken:

~~~
------------------------------------------------------------------- Captured stderr call --------------------------------------------------------------------
11:00:07.844 INFO processor.CalamariRecognize - INPUT FILE 0 / phys_0001
--------------------------------------------------------------------- Captured log call ---------------------------------------------------------------------
INFO     processor.CalamariRecognize:recognize.py:81 INPUT FILE 0 / phys_0001
================================================================== short test summary info ==================================================================
FAILED test/test_recognize.py::test_recognize - requests.exceptions.MissingSchema: Invalid URL 'OCR-D-IMG/INPUT_0017.tif': No scheme supplied. Perhaps you...
FAILED test/test_recognize.py::test_recognize_should_warn_if_given_rgb_image_and_single_channel_model - requests.exceptions.MissingSchema: Invalid URL 'OC...
FAILED test/test_recognize.py::test_word_segmentation - requests.exceptions.MissingSchema: Invalid URL 'OCR-D-IMG/INPUT_0017.tif': No scheme supplied. Per...
FAILED test/test_recognize.py::test_glyphs - requests.exceptions.MissingSchema: Invalid URL 'OCR-D-IMG/INPUT_0017.tif': No scheme supplied. Perhaps you me...
==================================================================== 4 failed in 16.04s =====================================================================
make: *** [Makefile:77: test] Error 1
~~~

Observations:

The new code from @bertsky's change in https://github.com/OCR-D/ocrd_calamari/commit/1f0252d0d7d1cffe76bc1f3626a536fe84106eff should download `OCR-D-IMG/INPUT_0017.tif` but doesn't:
~~~
% ls /tmp/test-ocrd-calamari/OCR-D-IMG 
OCR-D-IMG_0001.tif  OCR-D-IMG_0002.tif
~~~

--- a/test/test_recognize.py
+++ b/test/test_recognize.py
@@ -28,7 +28,12 @@ def workspace():
     os.makedirs(WORKSPACE_DIR)
 
     resolver = Resolver()
-    workspace = resolver.workspace_from_url(METS_KANT, dst_dir=WORKSPACE_DIR)
+    # due to core#809 this does not always work:
+    #workspace = resolver.workspace_from_url(METS_KANT, dst_dir=WORKSPACE_DIR)
+    # workaround:
+    shutil.rmtree(WORKSPACE_DIR)
+    shutil.copytree(os.path.dirname(METS_KANT), WORKSPACE_DIR)
+    workspace = resolver.workspace_from_url(os.path.join(WORKSPACE_DIR, 'mets.xml'))
 
     # The binarization options I have are:
     #

